### ISSUE TITLE
Cann't Show status in PowerShell 5.0 on Window 7

### ISSUE THREAD

**Issue by Nonlone** [dahlbyk/posh-git#360]
just like issue #161 
I update PowerShell to 5.0 with Managmant Framework and I installed the posh-git in command 'Install-Module posh-git -Scope CurrentUser' but it doesn't show the git status in PowerShell

**Comment by rkeithhill** [comment]
The version of posh-git on the PSGallery hasn't been updated in a while.  With posh-git 0.7.0 (coming soon) prompt modification will be automatic for most folks that import the module (unless you have already overridden the prompt function).  or now, look at the README.md.  For now, I recommend you do this.  First, from PowerShell execute `notepad $PROFILE` to edit (or create) your PowerShell profile.  This file is to PowerShell what .bashrc is to Bash.

In notepad, stick in these lines:
```powershell
Import-Module posh-git
function prompt {
    $origLastExitCode = $LASTEXITCODE
    Write-Host $ExecutionContext.SessionState.Path.CurrentLocation -NoNewline
    Write-VcsStatus
    $LASTEXITCODE = $origLastExitCode
    "> "
}
```
For more info on this, please see the [Using posh-git](https://github.com/dahlbyk/posh-git/blob/master/README.md#using-posh-git) section of the README file.

**Comment by Nonlone** [comment]
Thank you a lots! It work nice

**Comment by dahlbyk** [comment]
Going to go ahead and close this as fixed by #349. Keep an eye on #327 for the release.
